Common/Securities/IMarginCallModel.cs

```diff
@@ -46,6 +57,12 @@ public static class MarginCallModel
 
         private sealed class NullMarginCallModel : IMarginCallModel
         {
+            public SubmitOrderRequest GenerateMarginCallOrder(Security security, decimal netLiquidationValue, decimal totalMargin,
```

Comment:
If we leave the call site as a loop on holdings, probably we don't need multiple orders:
https://github.com/QuantConnect/Lean/pull/679/files#diff-ea81a360172dd59a9b5b98d829e4f26fR575

Maybe a better generalization (for another PR) could be replacing

`SubmitOrderRequest GenerateMarginCallOrder(Security security, decimal netLiquidationValue, decimal totalMargin, decimal maintenanceMarginRequirement);`

with

`List<SubmitOrderRequest> GenerateMarginCallOrders(SecurityPortfolioManager portfolio);`

?